app/src/main/res/layout/dialog_buy_buttons_payment_methods.xml

```diff
@@ -18,7 +18,6 @@
       android:layout_height="@dimen/button_height"
       android:layout_gravity="center_vertical"
       android:layout_marginEnd="@dimen/big_margin"
-      android:maxWidth="126dp"
       android:minWidth="80dp"
```

Comment:
Why did we removed the maxSize? If this is removed, for buttons with large text, the button may grow indefinitely.